### ISSUE TITLE
QF Donors need to know which projects they already donated to.

### ISSUE THREAD

**Issue by GriffGreen** [Giveth/giveth-dapps-v2#2958]
**As a QF Donor, I want to know that I have already donated to a project, so that I don't donate twice, unless I want to**

An important difference with gitcoin and us is that there is a cart feature in gitcoin, where for us we might have people accidentally (and intentionally) make multiple donations. 

We might consider a simple way of highlighting the projects that a donor already donated to on the QF page...

I'm imagining a check mark or something over the picture or something really obvious.... not sure how much backend work this would need...

Also it might be interesting to make make a highlighted text  on the donation page that says something like: "You have already donated to this project so the estimated matching will be incorrect."

I would want to do both honestly.. but not sure what the technical lift would be, also don't know if there is a better easier solution.

**Comment by mosaeedi** [comment]
### Project detail page
On the project detail page I suggest to let user know they are donated but still invite them to donate again!
<img width="372" alt="Screenshot 2023-09-27 at 6 14 52 PM" src="https://github.com/Giveth/giveth-dapps-v2/assets/13708322/d6d98a6b-e20a-4b1b-901b-e16e0940b0cb">


### On the donation page,
if the project is in the QF round and the user has already donated, we show this message!

<img width="953" alt="Screenshot 2023-09-27 at 6 54 05 PM" src="https://github.com/Giveth/giveth-dapps-v2/assets/13708322/62536f9e-f88a-4cf2-aea0-a24224aa1fc7">


### Otherwise, 
We just let them know they are donated but still invite them to do it again.

<img width="1029" alt="Screenshot 2023-09-27 at 6 56 00 PM" src="https://github.com/Giveth/giveth-dapps-v2/assets/13708322/b1e43fc4-1e45-4f02-9013-54c770ba5e6b">

**Comment by laurenluz** [comment]
I think the 1st two pieces are good. The project detail text that says "Already donated! Donate once more", and the banner text on the donate page that says "the estimated matching will be incorrect."

**Comment by laurenluz** [comment]
if we can build this in by next, week's Optimism round, it would be nice-to-have

@alireza-sharifpour

**Comment by MoeNick** [comment]
@MohammadPCh @mohammadranjbarz may talk about backedn requirements of this issue.

**Comment by laurenluz** [comment]
looks like the backend has been merged now! @MohammadPCh - are you unblocked?

**Comment by MoeNick** [comment]
@MohammadPCh @alireza-sharifpour is it progress?

**Comment by MohammadPCh** [comment]
@MoeNick  Yes, It's in progress

**Comment by MohammadPCh** [comment]
@mosaeedi Could you please add the Figma link?

**Comment by mosaeedi** [comment]
Please find the design here -> [Figma Link](https://www.figma.com/file/C3HykACu0eNImhAqu0our8/Giveth-projects?type=design&node-id=4493-99968&mode=design&t=ElhX23PWbJnHGGyV-4)

**Comment by maryjaf** [comment]
> if the project is in the QF round and the user has already donated, we show this message!

Based on @laurenluz 's comment I think this message should be shown in donate page @MohammadPCh

**Comment by laurenluz** [comment]
Yes! So, to be more clear, we should go with this @maryjaf @MohammadPCh:

### Project detail page
On the project detail page I suggest to let user know they are donated but still invite them to donate again!
<img width="372" alt="Screenshot 2023-09-27 at 6 14 52 PM" src="https://github.com/Giveth/giveth-dapps-v2/assets/13708322/d6d98a6b-e20a-4b1b-901b-e16e0940b0cb">


### On the donation page,
if the project is in the QF round and the user has already donated, we show this message!

<img width="953" alt="Screenshot 2023-09-27 at 6 54 05 PM" src="https://github.com/Giveth/giveth-dapps-v2/assets/13708322/62536f9e-f88a-4cf2-aea0-a24224aa1fc7">

**Comment by maryjaf** [comment]
Thanks @MohammadPCh This problem has been resolved.
